modules/log6x-installing-the-logging-ui-plug-in-cli.adoc

```diff
@@ -0,0 +1,52 @@
+:_newdoc-version: 2.18.4
```

Comment:
Make sure that this module file contains the required metadata, per the OCP doc guidelines: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata